src/components/MovieBlockItem/MovieBlockItem.tsx

```diff
@@ -34,19 +34,18 @@ const MovieBlockItem: FC<IMovieListItemProps> = ({
         <Poster src={TMDBImageHost + posterPath} alt="movie poster"></Poster>
         <ControlContainer>
           <MovieControl
-            movieId={id!}
-            index={index}
-            title={title!}
-            setMovies={setMovies}
+            movieId={id}
+            isFavourite={isFavourite}
+            controls={controls}
           />
         </ControlContainer>
       </MediaBlock>
       <TextInfo>
         <MovieHead title={title} releaseYear={releaseYear} yearSize="small" />
         <Footer>
-          {voteAverage && (
+          {voteCount! > 0 && (
```

Comment:
Please don't use non-null assertion operator (the short story, it's an antipattern and forbidden everywhere nowdays). In that case you can either assign default value or make the property non nullable.